BackEnd/README.md

```diff
@@ -0,0 +1,15 @@
+# PunORama
+
+I'm using the Linux native English dictionary (/usr/share/dict/american-english), to find puns.
```

Comment:
"I'm finding puns from words in the English dictionary which comes with Linux: `/usr/share/dict/american-english`.)"